src/pages/DatasetStatistics.tsx

```diff
@@ -113,61 +136,55 @@ export default function DatasetStatistics(props: DatasetStatisticsProps) {
           }
         </span>;
       default:
-        return <div/>;
+        return <span>N/A</span>;
     }
   };
 
-  if (!isLoading) {
+  if (!isLoading && datasetTerm) {
     return (
       <div style={{...Styles.PAGE, color: Theme.palette.primary}}>
         <div style={{justifyContent: 'space-between'}}>
           <div style={{marginTop: '25px'}}>
-            <div style={Styles.TITLE}>Dataset Statistics</div>
-            <div style={Styles.MEDIUM_ROW}>
-              <div style={{fontWeight: '500', marginRight: '5px'}}>Dataset ID:</div>
-              <div>{dataset?.datasetIdentifier}</div>
-            </div>
-            <div style={Styles.MEDIUM_ROW}>
-              <div style={{fontWeight: '500', marginRight: '5px'}}>Dataset Name:</div>
-              <div>
-                {extract('Dataset Name') || dataset?.name}
-              </div>
+            <div style={{fontSize: 20, fontWeight: 600}}>
+              <div>{datasetTerm.datasetIdentifier} - {datasetTerm.datasetName}</div>
             </div>
-            <div style={{paddingTop: '20px', paddingLeft: '30px'}}>
+            <LabeledField label={'Study'}>
+              {datasetTerm.study.studyName}
+            </LabeledField>
+            <LabeledField label={'Access Type'}>
               {accessInstructions()}
+            </LabeledField>
+            {(datasetTerm.accessManagement === AccessManagement.CONTROLLED || datasetTerm.accessManagement === AccessManagement.EXTERNAL) &&
+                <LabeledField label={'Data Use'}>
+                  {createDataUseDisplay({dataset: datasetTerm, tooltipPlace: 'right'})}
+                </LabeledField>
+            }
+            <LabeledField label={'Data Location'}>
+              {getDataLocationLink(datasetTerm.dataLocation, datasetTerm.url)}
+            </LabeledField>
+            <LabeledField label={'Phenotype'}>
+                {datasetTerm.study?.phenotype ?? 'N/A'}
+            </LabeledField>
+            <LabeledField label={'Participants'}>
+                {datasetTerm.participantCount}
+            </LabeledField>
+            <LabeledField label={'Principal Investigator(s)'}>
+                {datasetTerm.study.piName}
```

Comment:
The "(s)" entails a study can have more than one PI.  Is that indeed possible?

If so, then the template value (right now an always-singular `datasetTerm.study.piName`) should presumably account for possibly having multiple PIs.  

If not -- which AIUI is the case -- then `Principal Investigator(s)` should be `Principal Investigator`.

```suggestion
            <LabeledField label={'Principal Investigator'}>
                {datasetTerm.study.piName}
```